index.js

```diff
@@ -386,7 +394,7 @@ function fastifyView (fastify, opts, next) {
     // append view extension
     page = getPage(page, type)
     const requestedPath = getRequestedPath(this)
-    getTemplate(page, (err, template) => {
+    getTemplate(page, async (err, template) => {
```

Comment:
`getTemplate` should be ported to be an async function

Reply:
You should be using https://nodejs.org/api/fs.html#fspromisesreadfilepath-options.

(Available in all supported Node.js versions as `require('fs').promises.readFile`

Reply:
Happy to use it <3